guides/common/modules/proc_adding-permissions-to-a-role.adoc

```diff
@@ -10,12 +10,11 @@ To use the CLI instead of the {ProjectWebUI}, see the xref:cli-adding-permission
 . Select the *Resource type* from the drop-down list.
 The _(Miscellaneous)_ group gathers permissions that are not associated with any resource group.
 . Click the permissions you want to select from the *Permission* list.
-. Depending on the *Resource type* selected, you can select or deselect the *Unlimited* and *Override* checkbox.
-The *Unlimited* checkbox is selected by default, which means that the permission is applied on all resources of the selected type.
-When you disable the *Unlimited* checkbox, the *Search* field activates.
+. For certain *Resource type* options, the *Unlimited* checkbox is available.
+If *Unlimited* is selected, the permission is applied on all resources of the selected type.
```

Comment:
We might want to add that unlimited will be disabled if the role parent has a loc/org selected, alert in foreman says: "Info alert:The filter is scoped to the selected organizations and locations, therefore can't be unlimited"